test/ZendeskApi_v2.Test/HelpCenter/SectionTests.cs

```diff
@@ -45,6 +46,46 @@ public void CanGetSections()
             Assert.AreEqual(res1.Section.Id, res.Sections[0].Id.Value);
         }
 
+        [Test]
+        public void CanGetSectionsPaged()
```

Comment:
Please, Use the ```Assert.That ``` syntax and assert that the deletes complete

Reply:
Sorry, I Worded my request funny. 

I need all Asserts to use the ```Assert.That``` syntax, not just the deletes